src/containerapp/HISTORY.rst

```diff
@@ -9,6 +9,8 @@ upcoming
 * 'az containerapp env dapr-component init': support initializing Dapr components and dev services for an environment
 * 'az containerapp patch apply': support image patching for java application
 * Upgrade api-version to 2023-08-01-preview
+* Remove the region check for the Cloud Build feature in 'az containerapp up'/'az containerapp create' and 'az containerapp update' commands
+* Improve logs on the local buildpack source to cloud flow.
```

Comment:
Please add related commands in front of `Improve logs on the local buildpack source to cloud flow`.

Reply:
Thanks, done